.gitignore

```diff
@@ -16,6 +16,7 @@
 !\.gitignore
 !\.python-version
 !\.style.yapf
+!\.yamllint
```

Comment:
@wrowe - note that the ignore is prefixed with `!` - ie unignore

Reply:
Doh, thanks.